docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -104,19 +105,23 @@ If your module is set to :ref:`bypass arming on DroneCAN <dronecan_arming_and_by
 CAN Bus Hardware Setup
 =========================
 Before attempting to communicate with the flight controller, make sure that the hardware for the CAN bus is set up correctly. CANH and CANL should be connected on the 
-flight controller and all of the motors into one bus. There should also be a 120 ohm termination resistor between CANH and CANL on the bus. Refer to the datasheet
-of your Vertiq module for details on the pinout.
+flight controller and all of the motors into one bus. There should also be a 120 ohm termination resistor between CANH and CANL on the bus. Refer to 
+your module's family page for pinout information.
+
+DroneCAN Integration with a PX4 Flight Controller
+======================================================
 
 .. _dronecan_px4_fc_configuration:
 
 Flight Controller Configuration
-================================
-.. warning:: Between version 1.13.2 and 1.13.3 of the PX4 firmware, changes were made that require additonal steps to configure DroneCAN properly on the flight controller.
-    Steps that changed between the versions will be specified in the following sections, but if you have previously integrated with Vertiq module with a PX4 flight controller
-    on v1.13.2 or earlier, be aware that when you update your flight controller your integration process may have to change slightly.
+--------------------------------------
+.. warning:: 
+    
+    As PX4 is receives consistent new versions, the steps shown below may not match those necessary for your version of PX4. We will point out some key version 

```

Comment:
The first sentence here says "As PX4 is receives...", should probably be "As PX4 is consistently updated" or something